qiskit/circuit/instruction.py

```diff
@@ -144,6 +144,16 @@ def __eq__(self, other):
 
         return True
 
+    def __repr__(self) -> str:
+        """Generates a representation of the Intruction object instance
+        Returns:
+            str: A representation of the Instruction instance with the name,
+                 number of qubits, classical bits and params( if any )
+        """
+        return "Instruction(name = '{}', num_qubits = {}, num_clbits = {}, params = {})".format(
```

Comment:
```suggestion
        return "Instruction(name='{}', num_qubits={}, num_clbits={}, params={})".format(
```
This looks good, but I think would benefit from being a little more compact. E.g.
```
Instruction(name='rx', num_qubits=1, num_clbits=0, params=[3.14159])
```
vs
```
Instruction(name = 'rx', num_qubits = 1, num_clbits = 0, params = [3.14159])
```

Reply:
Okay looks good, will update it today!